indico/modules/attachments/clone.py

```diff
@@ -94,6 +95,7 @@ def _clone_attachment_folder(self, old_folder, new_object):
         folder.populate_from_attrs(old_folder, folder_attrs)
         folder.acl_entries = clone_principals(AttachmentFolderPrincipal, old_folder.acl_entries,
                                               self._event_role_map, self._regform_map)
+        signals.attachments.folder_created.send(folder, user=flask.session.user, cloned=folder)
```

Comment:
This `cloned=folder` arg is strange in general; and calling this signal would result e.g. in the cloning being logged to the event log (which adds noise we do not want).

Reply:
or you can keep it private like it's, and adding return folder at the end...

Reply:
Would it help to have a `_folder_map` (like there's `_attachment_map`)? Then you could just register your own internal cloner that depends on this cloner and access the mapping between old and new folders..

Reply:
Not really, because we need the same folder cloning as the core (cloned with attachments and attributes...).
It's just that we added a new foreign key to our folder object, and as the original cloning skipped the FK, we need to get back the new folder clones to update that FK.

Reply:
If `return folder` at the end of `_clone_attachment_folder` is sufficient for you then I suggest to just do that... I think that's better than adding a (somewhat obscure) signal call here.

Reply:
That will be fine for us. Thanks.
I will update the PR.